src/components/core/geoChart/legends/PushpinSizeLegend.tsx

```diff
@@ -0,0 +1,37 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { formatLegendLabel } from "../../../../components/visualizations/utils/common";
+
+export interface IPushpinSizeLegendProps {
+    format?: string;
+    numericSymbols: string[];
+    minValue: number;
+    maxValue: number;
+}
+
+export default class PushpinSizeLegend extends React.PureComponent<IPushpinSizeLegendProps> {
+    constructor(props: IPushpinSizeLegendProps) {
+        super(props);
+    }
+
+    public render() {
+        const { minValue, maxValue, format, numericSymbols = [] } = this.props;
+        const diff: number = maxValue - minValue;
+        return (
+            <div className="pushpin-size-legend">
+                <div className="circles-list">
+                    <div className="circle-max-value">
+                        <div className="circle-min-value" />
+                    </div>
+                </div>
+                <div className="labels">
+                    <span>{this.displayLabel(maxValue, diff, format, numericSymbols)}</span>
+                    <span>{this.displayLabel(minValue, diff, format, numericSymbols)}</span>
+                </div>
+            </div>
+        );
+    }
+    private displayLabel(value: number, diff: number, format: string, numericSymbols: string[]): string {
+        return formatLegendLabel(value, format, diff, numericSymbols);
```

Comment:
these parameters should have the same order for better reading :)
Or we can remove this function, and use `formatLegendLabel` directly. 
What do you think?